container-startup

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+# Startup helper for pcw. This file makes the handling with pcw within containers easier
+# See usage() for a usage summary
+
+CMD=$1
+shift
+
+function usage() {
+	echo "pcw startup helper"
+	echo "Usage: $0 COMMAND"
+	echo "COMMANDS:"
+	echo "   help                    Print help message"
+	echo "   run                     Standart publiccloud-watcher run procedure"
+	echo "   migrate                 Run database migration"
+	echo "   createsuperuser         Create superuser. Requires --email and --username argument"
+	echo ""
+	echo "All other commands will be passed as arguments to 'manage.py'"
+}
+
+case "$CMD" in
+  "")
+  	usage
+  	exit 1
+  	;;
+  "help")
+  	usage
+  	exit 0
+  	;;
+  "-h")
+  	usage
+  	exit 0
+  	;;
+  "--help")
+  	usage
+  	exit 0
+  	;;
+  "run")
+    # Ensure the database directory exists
+    mkdir -p /data/db
+  	python3 manage.py migrate
+  	python3 manage.py runserver 0.0.0.0:8000
```

Comment:
have you read `DO NOT USE THIS SERVER IN A PRODUCTION SETTING. ` in description of runserver command ?

Reply:
Yes. This server runs within a container, so I don't see any other generic way of permitting access to the django webserver other than allowing any host. Since the container provides some isolation itself, this should also be ok to have as default.

If you have however an idea how to make this better, please go ahead :slightly_smiling_face:

Reply:
I am speaking about something different here . In vault-docker we not using `manage.py runserver` at all to start pcw container . instead we doing it with `uwsgi --enable-threads --socket :3000 --module webui.wsgi`